pkgs/development/tools/misc/src-cli/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+let
+  owner = "sourcegraph";
+  repo = "src-cli";
+in
+buildGoModule rec {
+  pname = "src-cli";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+    rev = "${version}";
+    sha256 = "sha256-mhQdmfObeR7obd+o3emBf1GXIqVZAZCFNQGsG1a06+I=";
+  };
+
+  vendorSha256 = "sha256-18i/jdESkbvYHGwiKVpLMP5tnfBgiYV+sspk1iUGjOo=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Command line interface to Sourcegraph";
+    homepage = "https://github.com/sourcegraph/src-cli";
```

Add the changelog to meta will allow tools (e.g., `nix-update`) to add the changelog entry to the commit.

```suggestion
    homepage = "https://github.com/sourcegraph/src-cli";
    changelog = "https://github.com/sourcegraph/src-cli/blob/${version}/CHANGELOG.md";
```